frontend/src/components/ModelBox/ModelConfigurationSlider.tsx

```diff
@@ -32,11 +32,9 @@ function ModelConfigurationSlider({
 	const supportText = `more info about ${config.name}`;
 
 	return (
-		<div>
+		<fieldset>
 			<div className="model-config-info">
```

Comment:
can we switch the `<div className="model-config-info>` and  `<legend>` around so that the legend comes right after the fieldset? that way the fieldset has a label. That would require some css adjustments as well to move the icon again in the right place. 
You can check the accessibility tab in firefox (right click, second option from below, "Inspect Accessibility Properties")  and look in the accessibility tree if the fieldset (shows up as "group" in the accessibility tree) has been labelled correctly.